### PR TITLE
Add authentication in the node tool and modify the node tools doc

### PR DESCRIPTION
**Pull request by neuyilan**
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Review by mychaow — APPROVED** [review]
LGTM

File: cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java [hunk 1]
```diff
@@ -72,7 +83,9 @@ private MBeanServerConnection connect() {
     try {
       String jmxURL = String.format(JMX_URL_FORMAT, host, port);
       JMXServiceURL serviceURL = new JMXServiceURL(jmxURL);
-      JMXConnector connector = JMXConnectorFactory.connect(serviceURL);
+      Map<String, Object> environment = new HashMap<>(1);
```
Comment (jt2594838) [inline cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java:86]: `Collections.singletonMap` may be enough in this case

File: cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java [hunk 2]
```diff
@@ -84,7 +97,7 @@ private MBeanServerConnection connect() {
     return mbsc;
   }
 
-  String nodeToString(Node node){
+  String nodeToString(Node node) {
     return String.format("%s:%d:%d", node.getIp(), node.getMetaPort(), node.getDataPort());
```
Comment (jt2594838) [inline cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java:101]: I think we can also add `clientPort` here.

File: docs/UserGuide/System Tools/NodeTool.md [hunk 3]
```diff
@@ -34,7 +37,10 @@ you can specify the IP address and port of the cluster during startup.
 IP is the IP (or hostname) of the node that you expect to connect to,
 and port is the JMX service port specified when the IoTDB cluster is started.
 
-The default values are 127.0.0.1 and 31999, respectively.
+The default values are `127.0.0.1` and `31999`, respectively.
+
+When JMX authentication service is enabled, the user name and password of JMX service should be specified. 
+The default values are `iotdb` and `passw!d`, respectively.
```
Comment (jt2594838) [inline docs/UserGuide/System Tools/NodeTool.md:43]: Add the option names (-u and -pw) here.

**Review by jt2594838 — APPROVED** [review]

File: docs/UserGuide/System Tools/NodeTool.md [hunk 4]
```diff
@@ -34,7 +37,10 @@ you can specify the IP address and port of the cluster during startup.
 IP is the IP (or hostname) of the node that you expect to connect to,
 and port is the JMX service port specified when the IoTDB cluster is started.
 
-The default values are 127.0.0.1 and 31999, respectively.
+The default values are `127.0.0.1` and `31999`, respectively.
+
+When JMX authentication service is enabled, the user name and password of JMX service should be specified.
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:42]: These two lines should be put after the next two lines? And it's better to point out how to change authentication parameters, such as -u,-pw.

File: docs/zh/UserGuide/System Tools/NodeTool.md [hunk 5]
```diff
@@ -25,12 +25,17 @@ IoTDB集群版为您提供了NodeTool Shell工具用于监控指定集群的工
...
+
 # 使用说明
 
 ## 运行方式
+
 NodeTool Shell工具启动脚本位于$IOTDB_CLUSTER_HOME/sbin文件夹下，启动时可以指定集群运行的IP和PORT。
+其中IP为您期望连接的节点的IP，PORT为IoTDB集群启动时指定的JMX服务端口号，分别默认为`127.0.0.1`和`31999`。
 
-其中IP为您期望连接的节点的IP，PORT为IoTDB集群启动时指定的JMX服务端口号，分别默认为127.0.0.1和31999。
+对于开启了JMX鉴权服务的，启动时候需要指定JMX服务的用户名和密码，默认分别为`iotdb`和`passw!d`
```
Comment (LebronAl) [inline docs/zh/UserGuide/System Tools/NodeTool.md:38]: same as above

**Review by LebronAl — APPROVED** [review]
Great work! Only minor issues.

File: cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java (hunk 1 above)
Comment (neuyilan) [reply cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java:86]: Thanks for your remind, i'll change it

File: cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java (hunk 2 above)
Comment (neuyilan) [reply cluster/src/main/java/org/apache/iotdb/cluster/utils/nodetool/function/NodeToolCmd.java:101]: ok

File: docs/UserGuide/System Tools/NodeTool.md (hunk 3 above)
Comment (neuyilan) [reply docs/UserGuide/System Tools/NodeTool.md:43]: sure, i'll add the options

File: docs/UserGuide/System Tools/NodeTool.md (hunk 4 above)
Comment (neuyilan) [reply docs/UserGuide/System Tools/NodeTool.md:42]: sure, thanks

**Review by fanhualta — APPROVED** [review]

**Review by jt2594838 — APPROVED** [review]

**Pull request merged**